spec/factories/post.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :post do
+    caption "nice music noises"
+    image Rack::Test::UploadedFile.new(Rails.root + 'spec/files/images/maccabees.jpg', 'image/jpg')
```

Comment:
Metrics/LineLength: Line is too long. [99/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)